src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="form"
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="get"
+        spellcheck="false"
+        data-qa="big"
+      >
+        <label for="search" class="label">
+          <img
+            class="img"
+            src="/images/Search.svg"
+            alt="search icon"
+          >
+        </label>
+        <input
+          id="search"
+          class="input"
+          name="bigSearch"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </form>
+      <form
+        class="form form-small"
```

Comment:
If you are using as modifiers use this syntax
```suggestion
        class="form form--small"
```

Apply changes all over the document